.github/workflows/cloud-regress.yml

```diff
@@ -0,0 +1,104 @@
+name: Cloud Regression Test
+on:
+  push:
+    branches: [amasterov/regress-arm]
```

Comment:
I don't know how to write workflow yamls, but I guess this must be changed to mention `main` / `release` branch before merge?

Reply:
I will remove this before merging, thanks

Reply:
Removed